tutorials/enricher/pom.xml

```diff
@@ -13,9 +13,9 @@
 
   <properties>
     <!-- change this if you desire to use a different version of Apex Core -->
-    <apex.version>3.4.0</apex.version>
+    <apex.version>3.6.0-dt20161215</apex.version>
```

Comment:
Using that version for testing and validation is OK but these examples are supposed to be usable by anyone, not just RTS users. After testing, what gets committed should be a publicly accessible version. If someone tries to build with that apex version they get an error:

[ERROR] Failed to execute goal on project enricher: Could not resolve dependencies for project com.example:enricher:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: org.apache.apex:apex-common:jar:3.6.0-dt20161215, org.apache.apex:apex-engine:jar:3.6.0-dt20161215: Could not find artifact org.apache.apex:apex-common:jar:3.6.0-dt20161215 in central (https://repo.maven.apache.org/maven2) -> [Help 1]